src/unk_0206B910.c

```diff
@@ -0,0 +1,254 @@
+#include "global.h"
```

Comment:
no header file?
not every function is static so declaration should be in the header and included

Reply:
It's already present (`unk_0206B910.h`)

Reply:
ok